docs/arcticdb/ARCTICDB_INTEGRATION.md

```diff
@@ -96,9 +115,13 @@ And now you'll see only the columns you've had locked (we've locked no columns i
 
 ![](https://raw.githubusercontent.com/aschonfeld/dtale-media/master/images/arcticdb/demo/arcticdb_symbol5_col100.png)
 
-### Differences in Functionality from the Original D-Tale
+## Going Further with ArcticDB
+
+See the ArcticDB [docs](https://docs.arcticdb.io/) and [website](https://arcticdb.io/) for more information about it. The LMDB backend for ArcticDB is recommended for local testing, but for real workloads we recommend using an S3 backend. You can simply point D-Tale at an S3 backed ArcticDB instance and browse the data being written by processes across your system.
+
+## Differences in Functionality from the Original D-Tale
```

Comment:
@aschonfeld is there any way to just do a straight read from `ArcticDB` and hold the df in memory? I assume that wouldn't suffer from these limitations.

Reply:
@poodlewars if you don't set `use_store=True` then it will do exactly that.  A one-time read of everything for that node from arcticdb and then store it in memory.  But you also won't have the nice functionality of being able to navigate arcticdb and search for nodes.  So an example of doing this from the example I gave would be:

`dtale.show_arcticdb(uri='lmdb:///[path to DB]/test_db', library='lib1', symbol='symbol1')`

Reply:
The other option is to just do the read yourself and pass in the resulting dataframe:
```
import dtale
from arcticdb import Arctic

conn = Arctic('lmdb:///[path to DB]/test_db')
lib = conn['lib1']
df = lib.read('symbol1').data
dtale.show(df)
```